src/main/java/racing/domain/RacingGameManager.java

```diff
@@ -3,32 +3,27 @@
 import racing.view.InputView;
 import racing.view.ResultView;
 
-import java.util.ArrayList;
-import java.util.List;
 import java.util.Random;
 
 public class RacingGameManager {
     private static final Random random = new Random();
     private static final int RANDOM_MAX_NUMBER = 10;
 
     public void play() {
-        Count carCount = new Count(InputView.askNumberOfCarParticipants());
+        RacingCars racingCars = new RacingCars(InputView.askNumberOfCarParticipants());
         Count attemptCount = new Count(InputView.askNumberOfAttempt());
-        List<RacingCar> racingCars = new ArrayList<>();
-        for (int i = 0; i < carCount.getValue(); i++) {
-            racingCars.add(new RacingCar());
-        }
 
         ResultView.printGameResultTitle();
-        for (int i = 0; i < attemptCount.getValue(); i++) {
+        for (int i = 0; i < attemptCount.value(); i++) {
             playRound(racingCars);
         }
 
+        ResultView.printWinners(racingCars);
     }
 
-    private void playRound(List<RacingCar> racingCars) {
-        racingCars.forEach(this::playCar);
-        ResultView.printCarCurrentPosition(racingCars);
+    private void playRound(RacingCars racingCars) {
+        racingCars.getRacingCarList().forEach(this::playCar);
```

Comment:
객체를 단순한 자료구조로 사용하기 보다는 객체에게 메시지를 보내보아요~

랜덤에 대한 다양한 방법이 있겠죠!
하지만, racingCars은 랜덤에 대해 궁금해하지 않을것 같고 알게하고 싶지 않네요 🤔 
```suggestion
        racingCars.자동차_갯수만큼_배열을줄께_판단해서_움직여줘([4,5,6,2..])
```